### ISSUE TITLE
OSV .catch with url_option=1 ConnectionError

### ISSUE THREAD

**Issue by spatialtrail** [johntruckenbrodt/pyroSAR#141]
Hi, am attempting to use pyroSAR.S1.OSV().catch() to search for orbit files but getting a connection error:

```
from pyroSAR.S1 import OSV
orbit_files = OSV().catch(sensor=['S1A', 'S1B'], osvtype='POE', start='20210301T100926', stop='20210430T100926', url_option=1)
```

The error:

```
Traceback (most recent call last):
  File "C:\Users\spatialtrail\.conda\envs\s1_proc\lib\site-packages\IPython\core\interactiveshell.py", line 3418, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-2-0a28e7ec2258>", line 2, in <module>
    OSV().catch(sensor=['S1A', 'S1B'], osvtype='POE', start='20210301T100926', stop='20210430T100926', url_option=1)
  File "C:\Users\spatialtrail\.conda\envs\s1_proc\lib\site-packages\pyroSAR\S1\auxil.py", line 437, in catch
    items = self.__catch_gnss(sensor, osvtype, start, stop)
  File "C:\Users\spatialtrail\.conda\envs\s1_proc\lib\site-packages\pyroSAR\S1\auxil.py", line 391, in __catch_gnss
    subquery_response = requests.get(subquery, auth=auth, timeout=self.timeout)
  File "C:\Users\spatialtrail\.conda\envs\s1_proc\lib\site-packages\requests\api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "C:\Users\spatialtrail\.conda\envs\s1_proc\lib\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Users\spatialtrail\.conda\envs\s1_proc\lib\site-packages\requests\sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\spatialtrail\.conda\envs\s1_proc\lib\site-packages\requests\sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\spatialtrail\.conda\envs\s1_proc\lib\site-packages\requests\adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='dhusfeed.dhus.onda-dias.net', port=443): Max retries exceeded with url: /gnss/search?q=producttype:AUX_POEORB%20platformname:Sentinel-1%20beginPosition:%5B2021-03-01T10:09:26Z%20TO%202021-04-30T10:09:26Z%5D%20endPosition:%5B2021-03-01T10:09:26Z%20TO%202021-04-30T10:09:26Z%5D&start=0&rows=100&format=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001484B8D4850>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))
```

At first I just thought it was because https://scihub.copernicus.eu/gnss was down for maintenance but I can search & download successfully from there. So looking at the actual error I'm confused as to why its trying https://dhusfeed.dhus.onda-dias.net/ which does appear to be down.... Any ideas?!

**Comment by avalentino** [comment]
Maybe it is connected to the change in the API URL (see the related [news](https://scihub.copernicus.eu/news/News00868)).
Anyway I was not able to make it work with the new URL so honestly I'm confused as well.

**Comment by spatialtrail** [comment]
Yea or their larger news item: https://scihub.copernicus.eu/news/News00865 but I'm surprised there are no users reporting issues on https://forum.step.esa.int/

pyroSAR.snap.util.geocode is also broken because of this

**Comment by johntruckenbrodt** [comment]
Hi @spatialtrail thanks a lot for reporting this. Looks like the API has changed again, quite annoying. @avalentino thanks for making me aware of the URL change. However I think this only applies to Sentinel data download, the URL for the OSV files is https://scihub.copernicus.eu/gnss/. 

`S1.OSV.catch` is successful with the first API query with this URL:
https://scihub.copernicus.eu/gnss/search/?q=producttype:AUX_POEORB%20platformname:Sentinel-1%20beginPosition:[2021-03-01T10:09:26Z%20TO%202021-04-30T10:09:26Z]%20endPosition:[2021-03-01T10:09:26Z%20TO%202021-04-30T10:09:26Z]&format=json

In the JSON return one is now directed to dhusfeed.dhus.onda-dias.net for download of the actual files. It seems this does not work though. I'll see what I can do.

**Comment by johntruckenbrodt** [comment]
This should now be fixed with commit bfd235b. Pretty dirty hack though ;-)
